onyo/lib/onyo.py

```diff
@@ -563,49 +564,73 @@ def validate_anchors(self) -> bool:
 
         return True
 
-    def get_asset_paths(self,
-                        include: Iterable[Path] | None = None,
-                        exclude: Iterable[Path] | Path | None = None,
-                        depth: int = 0
-                        ) -> List[Path]:
-        r"""Select all assets in the repository that are relative to the given
+    def get_item_paths(self,
```

Comment:
In general, seeing all the logic in this method, I wonder if this method should be retired in favor of `get_items()` (which has not yet been converted) and have users query `[onyo.path.*]` according to their needs.

That very well be out of the scope of this PR, but I wanted to mention it.

Reply:
Not sure how it really should be done, but I agree that this probably should be done differently.
Just not an important part of switching over to the new powers.